js/storage.js

```diff
@@ -0,0 +1,34 @@
+const firebase = require('firebase')
+const uuidv4 = require('uuid/v4')
+
+export function initDatabase () {
+  firebase.initializeApp({
+    apiKey: 'AIzaSyDtCksjwncWyhTsZkMkIzct--e-lo3YHZU',
+    authDomain: 'plate-tectonics-3d.firebaseapp.com',
+    databaseURL: 'https://plate-tectonics-3d.firebaseio.com',
+    projectId: 'plate-tectonics-3d',
+    storageBucket: 'plate-tectonics-3d.appspot.com',
+    messagingSenderId: '89180504646'
+  })
+}
+
+export function saveModelToCloud (serializedModel, startCallback, endCallback) {
+  var db = firebase.database()
+  var uuid = uuidv4()
+
+  db.ref('models/' + uuid).set({
+    model: serializedModel
+  }, function () {
+    endCallback(uuid)
+  })
+  startCallback(uuid)
```

Comment:
It's synchronous, so I don't think `startCallback` is necessary. It can make calls to this func a bit easier to read.